modules/js/flappingfunctions.js

```diff
@@ -0,0 +1,239 @@
+const kInputStrokeStyle = "#FF3318";
+const kInputLineWidth = 4;
+function flapBeamWidthInputL(value) {
+  if (document.getElementById("flapBeamWidthL")) {
+    module.flapBeamWidthL = parseInt(value);
+    compositeArray[3].bodies[0].parts[1].render.strokeStyle = kInputStrokeStyle;
+    compositeArray[3].bodies[0].parts[1].render.lineWidth = kInputLineWidth;
+  }
+}
+function flapBeamWidthInputR(value) {
+  if (document.getElementById("flapBeamWidthR")) {
+    module.flapBeamWidthR = parseInt(value);
+    compositeArray[2].bodies[0].parts[1].render.strokeStyle = kInputStrokeStyle;
+    compositeArray[2].bodies[0].parts[1].render.lineWidth = kInputLineWidth;
+  }
+}
+function flapBeamWidthInput(value) {
+  flapBeamWidthInputR(value);
+  flapBeamWidthInputL(value);
+}
+function flapHeightInputL(value) {
+  module.flapBeamHeightL = parseInt(value);
+  compositeArray[3].bodies[0].parts[2].render.strokeStyle = kInputStrokeStyle;
+  compositeArray[3].bodies[0].parts[2].render.lineWidth = kInputLineWidth;
+}
+function flapHeightInputR(value) {
+  module.flapBeamHeightR = parseInt(value);
+  compositeArray[2].bodies[0].parts[2].render.strokeStyle = kInputStrokeStyle;
+  compositeArray[2].bodies[0].parts[2].render.lineWidth = kInputLineWidth;
+}
+function flapHeightInput(value) {
+  flapHeightInputR(value);
+  flapHeightInputL(value);
+}
+function flapOffsetInput(value) {
+  module.flapBeamOffset = parseInt(value);
+  for (let j = 1; j < 3; j++) {
+    compositeArray[2].bodies[0].parts[j].render.strokeStyle = kInputStrokeStyle;
+    compositeArray[3].bodies[0].parts[j].render.strokeStyle = kInputStrokeStyle;
+    compositeArray[2].bodies[0].parts[j].render.lineWidth = kInputLineWidth;
+    compositeArray[3].bodies[0].parts[j].render.lineWidth = kInputLineWidth;
+  }
+}
+const kLengthStrokeStyle = "#666";
+const kLengthLineWidth = 2;
+const kFlapWindowWidth = 0.75 * 0.5;
+function flapConstraintLengthL(value) {
+  // change c value so that the distance calculated between beam and object is different
+  // c is used in the beforeUpdate function in the individual module js files
+  c = parseInt(value);
+  jointComposites[
+    jointComposites.length - 2
+  ].constraints[0].render.lineWidth = kLengthLineWidth;
+  jointComposites[
+    jointComposites.length - 2
+  ].constraints[0].render.strokeStyle = kLengthStrokeStyle;
+}
+function flapConstraintLengthR(value) {
```

Comment:
A little bit of a nit-pick, but I would add a line break before these function definitions to improve readability.  At some point, it might be nice to add a comment above them to describe what they do (I still find many of the names kind of opaque).